msi/GitChooseEnvironmentDlg.wxs

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
IMHO, we should get the install down to one click for a proper install of Git for Windows and let the more advanced users click through options. That's what I've loved about WixAdvancedUI (and wixstdba has it's own limited way of doing the same).